lib/server.rb

```diff
@@ -16,6 +16,28 @@
 # To contact SUSE about this file by physical or electronic mail,
 # you may find current contact information at www.suse.com
 
+# Rack::TryStatic was taken from the rack-contrib repository (https://github.com/rack/rack-contrib)
+module Rack
+  class TryStatic
+    def initialize(app, options)
+      @app = app
+      @try = ['', *options[:try]]
+      @static = ::Rack::Static.new(lambda { |_| [404, {}, []] },options)
+    end
+
+    def call(env)
+      orig_path = env['PATH_INFO']
+      found = nil
+      @try.each do |path|
+        resp = @static.call(env.merge!({'PATH_INFO' => orig_path + path}))
+        break if !(403..405).include?(resp[0]) && found = resp
+      end
+      found or @app.call(env.merge!('PATH_INFO' => orig_path))
```

Comment:
Use `||` instead of `or`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.